easybuild/easyconfigs/g/genometools/genometools-1.5.8-foss-2015b-no-cairo.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'genometools'
+version = '1.5.8'
+versionsuffix = '-no-cairo'
```

Comment:
The dependencies are placeholders for the moment - I'm having trouble installing Python via easybuild (due to #2261).

These dependencies are only required for one of `genometools`' subcommands (`AnnotationSketch`) and adding `cairo=no` disables `AnnotationSketch` and drops both dependencies (https://github.com/genometools/genometools/blob/master/INSTALL).

I hope to submit easyconfigs for `genometools` these dependencies included and thus have `AnnotationSketch` enabled. Do you think these `-no-cario` ones should be dropped altogether?